demo/demo.cpp

```diff
@@ -20,6 +20,8 @@ std::shared_ptr<wr::SceneGraph> scene_graph;
 
 std::shared_ptr<wr::TexturePool> texture_pool;
 
+int frame_graph = 1;
```

Comment:
Please revert the demo.cpp. I tested it and it works. No real need to keep this in demo. Would fit better in a test.